packages/linear-genome-view/src/LinearGenomeView/components/LinearGenomeView.tsx

```diff
@@ -489,6 +489,7 @@ const LinearGenomeView = observer((props: { model: LGV }) => {
   const style = (initialized
     ? {
         display: 'grid',
+        width: model.width,
```

Comment:
if this is specified then the individual track's scroll bars are not visible. I think it needs removing

Reply:
is there any reason why it was included?

Reply:
because I think we need to manually specify the widths of views in order to keep control of how they are displaying.

for example, if you wanted for some reason to have two LGVs side by side underneath some parent view, and you wanted to draw polygons on an overlay between them, you would need to know the exact width of each of the LGVs.  That means you need to either control it explicitly with MST, or you need to measure with some measuring library and somehow propagate that measurement out of the react view layer and into MST where you can compute layouts with it.

does that make sense?

Reply:
I'm feel like if this is the case, a containing div can specify a width and then the LGV should be able to autofit inside of that, or just use flexbox layout side by side and no widths need to be specified. It is clear that the track scrollbars do not show up when this is set, showing that is is not autofitting right

Reply:
hmm you might be right, I'll explore that and get back to you

Reply:
alright i just removed it